test/41-device-api-v1.spec.ts

```diff
@@ -178,7 +199,31 @@ describe('SupervisorAPI [V1 Endpoints]', () => {
 	});
 
 	describe('GET /v1/apps/:appId', () => {
+		it('does not return information for an application when there is more than 1 container', async () => {
```

Comment:
Nice, this goes more in line with the documentation here https://www.balena.io/docs/reference/supervisor/supervisor-api/#http-api-reference